src/battle_script_commands.c

```diff
@@ -7187,6 +7289,70 @@ static void Cmd_tryKO(void)
     }
 }
 
+//special OHKO logic for FLASH
+static void Cmd_tryKO_Flash(void)
+{
+    u8 holdEffect, param;
+
+    if (gBattleMons[gBattlerTarget].item == ITEM_ENIGMA_BERRY)
+    {
+       holdEffect = gEnigmaBerries[gBattlerTarget].holdEffect;
+       param = gEnigmaBerries[gBattlerTarget].holdEffectParam;
+    }
+    else
+    {
+        holdEffect = ItemId_GetHoldEffect(gBattleMons[gBattlerTarget].item);
+        param = ItemId_GetHoldEffectParam(gBattleMons[gBattlerTarget].item);
+    }
+
+    gPotentialItemEffectBattler = gBattlerTarget;
+
+    //Squirtle with Sick Shades Lt. Surge battle solution
+    if (gBattleMons[gBattlerTarget].ability == ABILITY_SICK_SHADES)
+    {
+        gMoveResultFlags |= MOVE_RESULT_MISSED;
+        gLastUsedAbility = ABILITY_SICK_SHADES;
+        gBattlescriptCurrInstr = BattleScript_SturdyPreventsOHKO;
+        RecordAbilityBattle(gBattlerTarget, ABILITY_SICK_SHADES);
+        if (VarGet(VAR_TEMP_START_EVENT_BATTLE) == 4) {
+            FlagSet(FLAG_TEMP_MID_BATTLE_EVENT);
+            DebugPrintf("Mid-battle-event: Surge - SICK SHADES activated!");
+        }
+    }
+    else
+    {
+        u16 chance;
+        
+        //move always hits without restrictions
+        chance = TRUE;
+                
+        if (chance)
```

Comment:
This will always be true, so I don't think you need the `if` statement or anything in the `else` block here

Reply:
true. this was just a lazy change. Cleaned it up now